components/HeaderAppBar/index.tsx

```diff
@@ -0,0 +1,104 @@
+// Things to fix: the color of the app bar is blue, but idk how to change that to make it white like
+// the figma example
+
+import * as React from 'react'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Toolbar from '@mui/material/Toolbar'
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import Menu from '@mui/material/Menu'
+import MenuIcon from '@mui/icons-material/Menu'
+import Container from '@mui/material/Container'
+import Avatar from '@mui/material/Avatar'
+import MenuItem from '@mui/material/MenuItem'
+import { blue } from '@mui/material/colors'
+
+const settings = ['Sign out']
+
+export default function HeaderAppBar() {
```

Comment:
pass in a prop here that is a function called `setDrawerOpen(status: boolean) => void`